tests/test_models.py

```diff
@@ -137,3 +137,11 @@ def test_model_pk():
 
         products = await Product.objects.all(name__icontains="T")
         assert len(products) == 2
+
+
+@async_adapter
+async def test_model_exists():
+    async with database:
+        await User.objects.create(name="Tom")
+        assert await User.objects.filter(name="Tom").exists()
+        assert not await User.objects.filter(name="Jane").exists()
```

Comment:
Let's use `is True` and `is False` in these test cases.

Reply:
Done!